test/modules/MetricsValidator/src/tests/ValidateDocumentedMetrics.cs

```diff
@@ -76,6 +76,28 @@ protected override async Task Test(CancellationToken cancellationToken)
                 }
             }
 
+            // The following metric should not be populated in a happy E2E path.
+            // We are going to make a list and remove them here to not consider them as a failure.
+            IEnumerable<string> acceptableDegeneratedMetrics = new HashSet<string>
+            {
+                // "edgeagent_direct_method_invocations_count", // BEARWASHERE -- Actually make this pass
```

Comment:
Let's not check in any BEARWASHERE comments. ;)

Yes, it should be easy to force this metric by calling "ping" on Edge agent back in `Metrics.ValidateMetrics()`, something like:

```csharp
    var agent = new EdgeAgent(this.runtime.DeviceId, this.iotHub);
    await agent.PingAsync(token);
```

Reply:
Can confirm that works! Thank you!